### DISCUSSION TITLE
How to resend activation email or delete not fully registered account?

### DISCUSSION THREAD

**Discussion by luafanti** [community/community#44190] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Is there any way to resend the activation email? If there is no such possibility, can I delete an account that is not registered?
Unfortunately, I do not have the first e-mail with the activation link (it has been deleted)?

**Comment by radar-blip** [comment, marked as answer]
For something like that, you can always reach out to [support](https://support.github.com/request) for help.

**Reply by Mikimouse222** [reply]
> For something like that, you can always reach out to [support](https://support.github.com/request) for help.

))++++

**Comment by SarahLauran** [comment]
The same happened with me when I was working on [Alkaline Water Delivery in Los Angeles by ADivineh2o]([https://adivineh2o.com/). I asked the team to contact support and they were kind enough to help me get out of the situation.

**Comment by bzkarim250** [comment]
**1.Resend activation email**: Some platforms will allow you to resend the activation email if you did not receive it or it was deleted. You can check the platform's help or support pages to see if this option is available, or you can contact their support team to request that the activation email be resent.

**2.Delete an account that is not registered:** Some platforms may not allow you to delete an account that is not registered, as it does not yet exist in their system. You can check the platform's help or support pages to see if this option is available, or you can contact their support team to inquire about this possibility.

**3.Register a new account:** If you are unable to resend the activation email or delete the account, you may need to register a new account using a different email address.
